livekit-plugins/livekit-plugins-cartesia/livekit/plugins/cartesia/tts.py

```diff
@@ -53,15 +55,27 @@ def __init__(
         language: str = "en",
         encoding: TTSEncoding = "pcm_s16le",
         voice: str | list[float] = TTSDefaultVoiceId,
+        speed: TTSVoiceSpeed | float | None = None,
+        emotion: list[TTSVoiceEmotion] | None = None,
```

Comment:
Nice, let's still allow str in case there is new ones 
```suggestion
        emotion: list[TTSVoiceEmotion | str] | None = None,
```